<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>870</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customplot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>1021</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exp_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>91</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter function of x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>191</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>select min &amp; max values of x</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>400</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="zoom_in_h_button">
       <property name="text">
        <string>Zoom In - horizontally</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoom_out_h_button">
       <property name="text">
        <string>Zoom Out - horizontally</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoom_in_v_button">
       <property name="text">
        <string>Zoom In - vertically</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoom_out_v_button">
       <property name="text">
        <string>Zoom Out - vertically</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>230</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="max_spin">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="min_spin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Function Plotter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Functions supported:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>601</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="text">
     <string>&lt;ul&gt;
  &lt;li&gt;addition (+), subtraction/negation (-), multiplication (*), division (/), &lt;br&gt;exponentiation (^) and modulus (%)&lt;/li&gt;
  &lt;li&gt;abs, acos, asin, atan, atan2, ceil, cos, cosh, exp, floor, ln, log, log10,&lt;br&gt; pow, sin, sinh, sqrt, tan, tanh, fac &lt;/li&gt;
  &lt;li&gt;Constants: pi, e&lt;/li&gt;
&lt;/ul&gt;&lt;ui&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>exp_input</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
